system/src/system_network_manager.cpp

```diff
@@ -757,7 +779,7 @@ int NetworkManager::enableInterface(if_t iface) {
     return 0;
 }
 
-int NetworkManager::disableInterface(if_t iface) {
+int NetworkManager::disableInterface(if_t iface, network_disconnect_reason reason) {
```

Comment:
The `reason` argument is not used anywhere in this method.

Reply:
Added a comment explaining the behavior.